app/models/parsers/cardinal_pbmc_count_parser.rb

```diff
@@ -44,11 +44,18 @@ def qc_data
       @qc_data ||=
         {}.tap do |qc_data|
           rows.each do |row|
-            qc_data[row[0]] = { viability: Unit.new(row[4]), live_cell_count: Unit.new(row[2], 'cells') }
+            qc_data[get_well_location(row[0])] = {
+              viability: Unit.new(row[4]),
+              live_cell_count: Unit.new(row[2], 'cells')
+            }
           end
         end
     end
 
+    def get_well_location(cell)
+      cell.try(:split, ':')[1]
```

Comment:
This seems to be tightly coupled to the implementation here: https://github.com/sanger/sequencescape/blob/develop/app/models/well.rb#L329

Would it be worth either putting both the implementations next to each other or putting a comment in both locations to make the association obvious to someone who decides to update one or the other in the future?